lib/utils/getUnitFromValueNode.js

```diff
@@ -16,21 +15,26 @@ module.exports = function(node /*: Object*/) /*: ?string*/ {
     return null;
   }
 
+  // Ignore non-word nodes
+  if (node.type !== "word") {
+    return null;
+  }
+
+  // Ignore non standard syntax
+  if (!isStandardSyntaxValue(node.value)) {
+    return null;
+  }
+
+  // Ignore HEX
+  if (node.value[0] === "#") {
+    return null;
+  }
+
+  // Remove non standard stuff
```

Comment:
Here interesting case, for example:
```
a { margin: calc(100% - #{$margin * 2pix}); }
``` 

Now we throw error on `pix`, but it is part of interpolation (i.e. non standard) and i think we should ignore content in/into non standard syntax, but i am not sure, need some feedback

Reply:
> i think we should ignore content in/into non standard syntax

Yep, I think it's best if we ignore things within interpolation. The change in behaviour will be non-breaking.

Reply:
@jeddy3 let's do it in other PR